demos/simple-menu.html

```diff
@@ -32,10 +34,23 @@
       }
 
       .demo-content {
-        position: relative;
```

Comment:
`position: relative` needs to be present here otherwise the `mdc-menu-anchor` won't be positioned correctly.